Core.Auditing/ITracker.cs

```diff
@@ -0,0 +1,10 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+namespace Microsoft.CloudMine.Core.Auditing
+{
+    public interface ITracker
+    {
+
```

Comment:
No method declarations in this interface?

Reply:
From what I understand, this is a new interface that will be extended by IAuditLogger and then other alerting classes that may get created soon. @kivancmuslu please correct me if I'm wrong

Reply:
removed it for now and will be implemented in the future

Reply:
I think we should discuss this.